### ISSUE TITLE
Transformers: autobots/decepticons black screen flickering

### ISSUE THREAD

**Issue by KR-17** [DS-Homebrew/nds-bootstrap#1353]
**Your console:** 2DS

**Launch Method:** SD card

**Version used:** release

**Game tested/used/applicable:** Transformers: Decepticons (USA)

**Expected Behavior:** Game playing normally without screen glitches

**Actual Behavior:** During 3D gameplay the upper screen flickers black constantly. Most of the time only half of the screen flickers, but sometimes the entire screen turns black for a split second and then goes back to normal.

### Steps to reproduce
1. Simply start playing the game after the title screen
2. 
3.

**Comment by RocketRobz** [comment]
Try updating nds-bootstrap to v0.54.1, and see if that fixes the flickering.

**Comment by KR-17** [comment]
Nope. Updated both twilight menu and nds-bootstrap and nothing's changed. as soon as the first FMV and the in-game cutscene ends, the game starts to flicker like hell.

**Comment by RocketRobz** [comment]
Try changing the `ARM9 CPU Speed` option then. If TWL speed is causing the issue, try reverting to NTR.

**Comment by KR-17** [comment]
> Try changing the `ARM9 CPU Speed` option then. If TWL speed is causing the issue, try reverting to NTR.

Did that too, but it doesn't seem to affect anything, the flickering still happens.

**Comment by spellboundtriangle** [comment]
Just tried it myself in 0.54.2 with Decepticons (USA rev 1). I wouldn't describe it as "constantly," but it's definitely noticeable (primarily while driving through the city area, haven't been anywhere else yet). It doesn't seem to occur at all on the real game card. It does occur on flashcard (with regular nds-bootstrap, B4DS, & EOS) more than nds-bootstrap via DSi SD.

**Comment by TNKRetro** [comment]
This happens to me as well on the lastest update of nds-bootstrap. The game's graphics flicker badly in some areas
